# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1455831046 28800
#      Thu Feb 18 13:30:46 2016 -0800
# Branch stable
# Node ID aaf59727f8095c086d115408b9fa858d83aa530c
# Parent  596e19e7e5717100c9f30a5631031043df3a82ac
doc: Use consistent spacing for memory/storage sizes

* README, doc/doxyhtml/README, install.txi, fact.m: Insert space between
value and size unit label.

diff --git a/README b/README
--- a/README
+++ b/README
@@ -30,17 +30,17 @@ may be found on Savannah (http://savanna
 under the Source Code tab.
 
 Installation
 ------------
 
 Octave requires approximately 1.4 GB of disk storage to unpack and
 compile from source (significantly less, 400 MB, if you don't compile
 with debugging symbols).  Once installed, Octave requires
-approximately 350MB of disk space (again, considerably less, 70 MB,
+approximately 350 MB of disk space (again, considerably less, 70 MB,
 if you don't build shared libraries or the binaries and libraries do
 not include debugging symbols).
 
 To compile Octave, you will need a recent version of GNU Make.  You
 will also need a recent version of g++ or another ANSI C++ compiler.
 You will also need a Fortran 77 compiler or f2c.  If you use f2c, you
 will need a script like fort77 that works like a normal Fortran
 compiler by combining f2c with your C compiler in a single script.
diff --git a/doc/doxyhtml/README b/doc/doxyhtml/README
--- a/doc/doxyhtml/README
+++ b/doc/doxyhtml/README
@@ -2,10 +2,10 @@ This directory contains documentation in
 Octave's source code.  It is not created by default.
 
 To produce Doxygen documentation use
 
 make doxyhtml
 
 Doxygen documentation can be helpful for developers of Octave, but is not
 needed by users of Octave.  In addition, the documentation requires
-approximately 1.2GB of storage space.  For these reasons it is not maintained
+approximately 1.2 GB of storage space.  For these reasons it is not maintained
 under version control nor distributed in tarballs.
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -599,22 +599,22 @@ Note: the following only applies to syst
 Configuring Octave with @option{--enable-64} cannot magically make a
 32-bit system have a 64-bit address space.
 
 On 64-bit systems, Octave is limited to (approximately) the following
 array sizes when using the default 32-bit indexing mode:
 
 @example
 @group
-double:         16GB
-single:          8GB
-uint64, int64:  16GB
-uint32, int32:   8GB
-uint16, int16:   4GB
-uint8, int8:     2GB
+double:         16 GB
+single:          8 GB
+uint64, int64:  16 GB
+uint32, int32:   8 GB
+uint16, int16:   4 GB
+uint8, int8:     2 GB
 @end group
 @end example
 
 In each case, the limit is really (approximately) @math{2^{31}} elements
 because of the default type of the value used for indexing arrays
 (signed 32-bit integer, corresponding to the size of a Fortran INTEGER
 value).
 
diff --git a/scripts/miscellaneous/fact.m b/scripts/miscellaneous/fact.m
--- a/scripts/miscellaneous/fact.m
+++ b/scripts/miscellaneous/fact.m
@@ -41,17 +41,17 @@ function truth = fact ()
       "Richard Stallman is so handsome that when he was young he was responsible for all other geeks not being able to get girls. This is why he has to cover his face with a thick layer of hair.";
       "Some people check their computers for viruses. Viruses check their computers for Richard Stallman.";
       "Richard Stallman memorises all his documents. In binary. He just types everything in whenever he needs a document.";
       "When Richard Stallman makes a sudo command, he loses permissions.";
       "Richard Stallman's beard is made of parentheses.";
       "Richard Stallman's DNA is in binary.";
       "Richard Stallman's nervous system is completely wireless.";
       "Richard Stallman's brain accepts UNIX commands.";
-      "If Richard Stallman has 1GB of RAM, and if you have 1GB of RAM, Richard Stallman has more RAM than you.";
+      "If Richard Stallman has 1 GB of RAM, and if you have 1 GB of RAM, Richard Stallman has more RAM than you.";
       "Richard Stallman eats ethernet cables. That's why they invented wireless.";
       "Richard Stallman has a katana. 'Nuff said.";
       "Richard Stallman wrote a program that divides by zero.";
       "Richard Stallman doesn't use zip drives; he just squeezes the hard drive.";
       "Richard Stallman's compiler is afraid to report errors.";
       "Richard Stallman wrote the compiler God used. The Big Bang was the Universe's first segfault.";
       "Richard Stallman successfully compiled a kernel of popcorn.";
       "Richard Stallman doesn't write programs; they write themselves out of reverence.";
